### ISSUE TITLE
"Absent" toggle doesn't work

### ISSUE THREAD

**Issue by vsalustiano-st** [Nain57/Smart-AutoClicker#551]
### Version of Smart AutoClicker

3.1.0-beta01 but I tried 3.0.6 as well

### Apk

Github

### Device brand and model

Poco F5 Pro

### Version of your Android OS

14

### Type

Smart scenario

### Describe what happened?

I started with a simple script that clicks on a button when an image is found on the screen. When I test this condition, I can see a green border around the scanned area, indicating that the match was a success. So far so good.

My problem is with doing the opposite: clicking on a button when an image is NOT FOUND. I tried the "Absent" toggle, but it just doesn't work. When I test this condition, the border on the scanned area remains red, regardless if the image is found or not. I don't know much yet, but I think something else is causing conditions to never match in case the "Absent" toggle is on.

### Describe what should have happened?

I would expect the border on the scanned area to become green and execute the linked actions in case the image is not found.

**Comment by Nain57** [comment]
Hi and thank you for your feedback.

I made some manual tests, and I have added a test case for this specific use case (https://github.com/Nain57/Smart-AutoClicker/commit/5828dd3c65bd14669d730c69dc132ec61cb66377), everything is working as intended.

I thinks this comes from a misunderstanding due to the UI. When testing an ImageCondition, the rectangle color is indicating if the condition image is detected on the screen, and not the fulfillment state of the condition itself. This means changing the value to absent wil not change the color of the rectangle during the testing, as it is still detected. But during normal execution, everything will behave as you expect.

This is more a minor UI issue, i have updated it to reflect the fulfillment state instead of the detection, so it will be red in your use case. 

Not a bug but UI change: Accepted. This can be misleading for new users, and it will provide a visual feedback for this feature.
Moving from:
![Capture d'écran 2024-11-23 150903](https://github.com/user-attachments/assets/48f8122e-25fe-4e85-b610-f69d9b02df5e)

To:
![Capture d'écran 2024-11-23 150923](https://github.com/user-attachments/assets/b14cb0e6-f273-4ca8-a66f-02ca54f0ec2a)
